src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1163,6 +1163,12 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 
 			// List failed plugin updates.
 			if ( ! empty( $failed_updates['plugin'] ) ) {
+
+				/*
+				* If this string changes, the `str_replace()` call in
+				* `WP_Rollback_Auto_Update::auto_update_rollback_message()`
+				* should also be changed.
+				*/
```

Comment:
```suggestion
				/*
				 * If this string changes, the `str_replace()` call in
				 * `WP_Rollback_Auto_Update::auto_update_rollback_message()`
				 * should also be changed.
				 */
```
Correct multiline comment formate.